src/uncontrast/pages/UploadNewPaper/ExperimentsSection/TargetStimuli.jsx

```diff
@@ -282,12 +301,38 @@ export default function TargetStimuli({
               </div>
             </div>
             <div className="border-r-2 border-blue h-24"></div>
-            <div id="trash+submit">
-              <TrashButton
-                fieldValues={fieldValues}
-                index={0}
-                handleDelete={handleDelete}
-              />
+            <div className="flex flex-col items-center gap-1">
+              <div id="trash+submit">
+                <TrashButton
+                  fieldValues={fieldValues}
+                  index={0}
+                  handleDelete={handleDelete}
+                />
+              </div>
+              {/* {disableCondition && (
+                <Tooltip animation content="Edit" trigger="hover">
+                  <button
+                    type="button"
+                    onClick={() => {
+                      enableEdit(index);
+                    }}>
+                    <Edit className="w-6 h-6" />
+                  </button>
+                </Tooltip>
+              )}
+              {!disableCondition && (
+                <SubmitButton
+                  submit={async () => {
+                    if (!editble) {
+                      handleSubmit(fieldValues);
+                    } else {
+                      const res = await handleEdit(fieldValues);
+                      res && enableEdit(false);
+                    }
+                  }}
+                  disabled={!submitCondition || disableCondition}
+                />
+              )} */}
```

Comment:
a lot of commented out jsx

Reply:
לקחתי רברס מהשינויים באנקונטראסט כי אני צריך לסגור קודם משהו עם מאור. לכן כל הקוד קומנטד ככה